Source/JavaScriptCore/assembler/MacroAssemblerX86_64.h

```diff
@@ -2137,21 +2137,63 @@ class MacroAssemblerX86_64 : public MacroAssemblerX86Common {
 
     void vectorReplaceLane(SIMDLane simdLane, TrustedImm32 lane, RegisterID src, FPRegisterID dest)
     {
-        m_assembler.pinsr(simdLane, lane.m_value, src, dest);
+        switch (simdLane) {
+        case SIMDLane::i8x16:
+            m_assembler.pinsrb(lane.m_value, src, dest);
```

Comment:
For register/register instructions, use `_rr` postfix.
Do that for all the other instructions.